examples/mqttc/mqttc_pub.c

```diff
@@ -174,7 +174,9 @@ static int initserver(const FAR struct mqttc_cfg_s *cfg)
         }
 
       close(fd);
+      fd = -1;
     }
+
   while ((itr = itr->ai_next) != NULL);
```

Comment:
Did nxstyle complain about this?  AFAIK a new line isn't required here.

Reply:
No, I just put back in place.
It complained about the missing line after the right brace before `close(fd)` and I extended the idea to this one.
But it is not the case since it's a "do while".